testing/tests/test_e2e/test_relations.py

```diff
@@ -88,6 +90,124 @@ def pre_event(charm: CharmBase):
     )
 
 
+@pytest.mark.parametrize(
+    ("original_data", "new_data", "result_data"),
+    [
+        (  # populate the relation data from scratch
```

Comment:
What about using these comments as the ID for the test? `test_relation_set[populate the relation data from scratch]` is a lot clearer than whatever auto ID this is going to end up as.